cedar-policy-core/src/parser/err.rs

```diff
@@ -180,11 +180,11 @@ pub enum ToASTErrorKind {
     #[error("this policy is missing the `{0}` variable in the scope")]
     MissingScopeConstraint(Var),
     /// Returned when a policy has an extra scope clause. This is not valid syntax
-    #[error("this policy has an extra constraint in the scope: `{0}`")]
+    #[error("this policy has an extra scope constraint in the scope: `{0}`")]
```

Comment:
redundant: "extra scope constraint in the scope"

Reply:
Yeah, that was removed in #825, but found its way back in there